oauth2_http/java/com/google/auth/oauth2/GoogleCredentials.java

```diff
@@ -36,9 +36,14 @@
 import com.google.api.client.json.JsonObjectParser;
 import com.google.api.client.util.Preconditions;
 import com.google.auth.http.HttpTransportFactory;
+import org.apache.commons.io.FileUtils;
```

Comment:
Could you figure out how to use Guava for this functionality instead of adding a second utility dependency?